### DISCUSSION TITLE
[Q&A] 如何设置插件市场索引

### DISCUSSION THREAD

**Discussion by dcityteg** [lobehub/lobehub#400] [Self Hosting] [answered]
### 🧐 问题描述 | Proposed Solution

![image](https://github.com/lobehub/lobe-chat/assets/132677224/cdcd7dba-c605-4384-b113-17a54c066e79)
提示说要在环境变量里写，请问一下变量名称是什么？

**Comment by arvinxx** [comment, marked as answer]
请查阅 https://github.com/lobehub/lobe-chat/wiki/Environment-Variable.zh-CN#plugins_index_url
